community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/LogFileInformation.java

```diff
@@ -24,25 +24,25 @@
 public interface LogFileInformation
 {
     /**
-     * @return the reachable transaction that is farthest back of them all, in any existing version.
+     * @return the reachable entry that is farthest back of them all, in any existing version.
      */
-    long getFirstExistingTxId() throws IOException;
+    long getFirstExistingId() throws IOException;
```

Comment:
Maybe include `Entity` in all method names? Otherwise it seems like `LogFileInformation` is all about ids and not log entries.